src/test/java/Homework16.java

```diff
@@ -0,0 +1,37 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+
+    @Test
+
+    public void registrationNavigation() {
+
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins");
+
+        WebDriver driver = new ChromeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/73595df4-c47e-4e1c-a881-4ab3ba604311)

HINT

 Add `--remote-allow-origins=*`
 
 
![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/d074972c-0f46-4a75-b109-3ff6fbb329b9)

 After the Chrome 111 update, you can no longer kick off a chromedriver instance unless you add an additional chrome option:
"--remote-allow-origins=*"

 ChromeOptions options = new ChromeOptions();
        options.addArguments("--disable-notifications","--remote-allow-origins=*", "--incognito","--start-maximized");
options.setExperimentalOption("excludeSwitches", new String[]{"enable-automation"});

  
Below are the list of available and most commonly used arguments for ChromeOptions class

**start-maximized:** Opens Chrome in maximize mode
**incognito:** Opens Chrome in incognito mode
**headless:** Opens Chrome in headless mode
**disable-extensions:** Disables existing extensions on Chrome browser
**disable-popup-blocking:** Disables pop-ups displayed on Chrome browser
**make-default-browser:** Makes Chrome default browser
**version:** Prints chrome browser version
**disable-infobars:** Prevents Chrome from displaying the notification ‘Chrome is being controlled by automated software